### DISCUSSION TITLE
large file size

### DISCUSSION THREAD

**Discussion by omega3** [rhasspy/piper#268] [Q&A] [answered]
I created mp3 from about 10 average sentences like this:
`cat sample.txt | piper -m en_GB-southern_english_female-low.onnx --length-scale 1 --sentence-silence 1 --output_file piper_output.mp3`
and file size is 1,7MiB.
If I processed a book, file size would be huge.

Is there anything that can decrease file size?

**Comment by synesthesiam** [comment, marked as answer]
The output is always a WAV file, even if you name it `.mp3`. You can convert on the fly to MP3 using `ffmpeg` like this:

```sh
cat sample.txt | piper -m en_GB-southern_english_female-low.onnx --output-raw | \
    ffmpeg -f s16le -ar 16000 -ac 1 -i 'pipe:' -f mp3 -q:a 0 piper_output.mp3
```

Adjust `-q:a 0` to change the quality level.
